src/extensions/timeseries/TimeseriesViewNew/TimeseriesWidgetNew.tsx

```diff
@@ -14,7 +14,7 @@ interface Props {
     y_scale_factor: number
     width: number
     height: number
-    leftPanels: any[]
+    visibleChannelIds?: number[] | null
```

Comment:
new optional prop called visibleChannelIds that controls which channels are visible